stabilityai/pytorch/inference/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -0,0 +1,27 @@
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference:2.0.1-gpu-py310-cu118-ubuntu20.04-sagemaker as sagemaker
```

Comment:
There seems to be something strange happening in this FROM statement. I don't have write access to your branch, but can you change this to capital "AS"